db/migrations/20220319173554_lsg/migration.sql

```diff
@@ -0,0 +1,11 @@
+-- CreateTable
```

Comment:
Why is this migration just called `lsg`?

Reply:
laziness